src/Apps/Feature/Components/FeatureSet.tsx

```diff
@@ -47,8 +53,43 @@ export const FeatureSet: React.FC<FeatureSetProps> = ({
         ]}
         gridGap={2}
       >
-        {children}
+        {set.orderedItems.edges.map(({ node: orderedItem }) => {
+          switch (orderedItem.__typename) {
+            case "FeaturedLink":
+              return (
+                <FeatureFeaturedLink
+                  key={orderedItem.id}
+                  featuredLink={orderedItem}
+                />
+              )
+            default:
+              // TODO: Unimplemented: Artist | Artwork | Gene | Sale | PartnerShow | Profile | OrderedSet
```

Comment:
@sweir27 this is what's currently supported

https://github.com/artsy/metaphysics/blob/2784f200a7c75843ab898dc7b575173f8d03668c/src/schema/v2/item.ts#L15-L25

For instance Genes are used on artists index pages.

Reply:
Which is weird weird in terms of restricting things — Sets aren't exclusively used for Feature pages, so support in this case becomes what's available in the UI rather than what's possible through the types.

Reply:
Conceptually it seems OK that we have the ability to create sets of many different types (because they are generic/can be used elsewhere), but the sets that are used _within_ Feature pages are constrained.

Agree that it should be an actual data model limitation, but for now maybe metaphysics can restrict?

Reply:
I concur — I sort of suspect we'll have to create a different OrderedItem type that is restricted to the types we want.

Reply:
So what do we want to restrict it to? We should compile a list of valid features that express the possibilities in staging.

Reply:
Yeah. I mean to do that ☝️ today 😄